pages/integrations/bitbucket.md.erb

```diff
@@ -2,46 +2,58 @@
 
 Buildkite integrates with [Bitbucket](https://bitbucket.org/) to provide automated builds based on your source control. You can run a build every time you push code to Bitbucket, and pull requests can have their build status live-updated as builds progress.
 
-<%= image "build-status.png", width: 1160/2, height: 512/2, alt: "Screenshot of the Bitbucket builds popup" %>
-
 This guide shows you how to set up your Bitbucket builds with Buildkite.
 
 {:toc}
 
 ## Set up the Bitbucket webhook
 
-Once you’ve created a pipeline in Buildkite and pasted in your bitbucket.org’s repository URL, Buildkite will show you setup instructions for configuring your Bitbucket webhooks.
+Once you’ve created a pipeline in Buildkite and copied in your bitbucket.org’s repository URL, Buildkite shows you setup instructions for configuring your Bitbucket webhooks.
 
-You can also find these instructions by following the "Bitbucket Setup Instructions" link on your Buildkite pipeline’s Settings page:
+You can also find these instructions by following the _Bitbucket Setup Instructions_ link on your Buildkite pipeline’s Settings page:
```

Comment:
Should we just delete this sentence? What does it add?

Reply:
Users are shown the instructions as a page that comes up during pipeline creation. This tells them how to get back to it if they skip the step/need to refer back to it.